### PR TITLE
updated the CKAN detection

### PR DESCRIPTION
**Pull request by seralf**
updated the CKAN record, adding metadata part

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -1366,6 +1366,9 @@
 				"Java",
 				"PostgreSQL\\;confidence:80"
 			],
+			"meta": {
+				"generator": "ckan [\\d]\\.[\\d]\\.[\\d]"
```
Comment (jvoisin) [inline src/apps.json]: This could be used to fingerprint the version. Care to add the necessary boilerplate to do it?

Comment (seralf) [inline src/apps.json]: I don't know how to add the fingerprint, are there any guideline to do it? (or if it's simple: can you do it?)

Comment (jvoisin) [inline src/apps.json]: Sure, I could do it if you want, but it's [specified]( https://github.com/AliasIO/Wappalyzer/wiki/Specification#patterns ) (although it lacks of good example), and you can grep for `version` in the `src/apps.json` file for examples, if you're planning to continue to contribute to wappalyzer. Worst case, I'll likely do it in a couple of days if you prefer :)

**Comment by seralf** [comment]
thank you for the reference, I'll try to do my best. Every time I've found
something not covered I've added, in the past, and of course I'd like to
continue contributing ;-)

2017-02-28 14:24 GMT+01:00 jvoisin <notifications@github.com>:

**Comment by seralf** [comment]
ok, I should have added the version, as suggested: please let me know if I've missed something, thanks

**Comment by jvoisin** [comment]
A closing double-quote ;)

**Comment by seralf** [comment]
d'oh, thanks... fixed! :-)

File: src/apps.json [hunk 2]
```diff
@@ -1356,8 +1356,8 @@
 				1
 			],
 			"headers": {
-				"Access-Control-Allow-Headers": "X-CKAN-API-KEY",
-				"Link": "<http://ckan.org/>; rel=shortlink"
+				"Access-Control-Allow-Headers": "X-CKAN-API-KEY\\;confidence:90",
```
Comment (jvoisin) [inline src/apps.json]: This should be 100%, shouldn't it?

Comment (jvoisin) [inline src/apps.json]: Please state the regexp with `^`.

**Comment by AliasIO** [comment]
Would you mind responding to @jvoisin's suggestions?

**Comment by seralf** [comment]
I've done a couple of changes, please see if they are ok for you now, thanks



2017-04-17 4:11 GMT+02:00 Elbert Alias <notifications@github.com>:

**Comment by seralf** [comment]
Hi all the errors seems to be fixed now, thanks both of you for the suggestions

File: src/apps.json [hunk 3]
```diff
@@ -1364,8 +1364,11 @@
 				"Python",
 				"Solr",
 				"Java",
-				"PostgreSQL\\;confidence:80"
+				"PostgreSQL"
 			],
+			"meta": {
+				"generator": "(?:)ckan(?:)"
```
Comment (gadcam) [inline src/apps.json]: Sorry but I don't understand : what is role of these groups <code>(?:)</code> ?
Moreover why didn't you remove the version detection ?

**Comment by seralf** [comment]
if there are better ways to write it, please edit it!
If the groups are not needed you can remove them. I've removed the version
because as far as I understand this part does not need it

2017-05-19 18:39 GMT+02:00 Camille Barneaud <notifications@github.com>:

**Comment by gadcam** [comment]
As for Virtuoso "I'm only a contributor so I can't remove anything.". I think these groups do nothing..

**Comment by DaAwesomeP** [comment]
"Contributor" means that at least one of your commits has been merged in the main repo. "Collaborator" means that you can push to repo, control issues, etc.

**Comment by seralf** [comment]
ok, I've made some modification based on the suggestions, please see if it can be merged, finally

**Pull request merged**